sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/display/DisplayDataEvaluatorTest.java

```diff
@@ -71,4 +71,23 @@ public void populateDisplayData(DisplayData.Builder builder) {
     assertThat(displayData, not(hasItem(hasDisplayItem("compositeKey", "compositeValue"))));
     assertThat(displayData, hasItem(hasDisplayItem("primitiveKey", "primitiveValue")));
   }
+
+  @Test
+  public void testPrimitiveTransform() {
+    PTransform<? super PCollection<Integer>, ? super PCollection<Integer>> myTransform = ParDo.of(
+        new DoFn<Integer, Integer>() {
+      @Override
+      public void processElement(ProcessContext c) throws Exception {}
+
+      @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        builder.add(DisplayData.item("foo", "bar"));
+      }
+    });
+
+    DisplayDataEvaluator evaluator = DisplayDataEvaluator.forRunner(DirectPipelineRunner.class);
```

Comment:
probably out of scope for this PR, but will need to be fixed at some point, as we plan to remove the DirectPipelineRunner: this probably shouldn't take a runner and instead should use TestPipeline#testingPipelineOptions(), and then runners can include any runner-specific tests in the appropriate runner-suite (populating the environment as appropriate)